### PR TITLE
doc: minor updated to macOS instructions

### PR DESCRIPTION
**Pull request by nashif**
Update macOS instructions and remove obsolete kconfig build steps.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

File: doc/getting_started/getting_started.rst [hunk 1]
```diff
@@ -231,13 +231,23 @@ without the Zephyr SDK`_ for details.
    and extract it on your file system. This example assumes the compiler was
    extracted to: :file:`<user folder>/gcc-arm-none-eabi-5_3-2016q1/`.
 
+   On macOS, the toolchain can also be installed using brew::
+
+   # to tap the repository
+   $ brew tap osx-cross/arm
```
Comment (andrewboie) [inline doc/getting_started/getting_started.rst]: the text box is really annoying to copy-paste into a terminal with the comments and the leading $, I'd like to see those removed.

**Review by tyrantlizard77 — APPROVED** [review]

File: doc/getting_started/installation_mac.rst [hunk 2]
```diff
@@ -57,11 +57,11 @@ Install tools to build Zephyr binaries:
 
 .. code-block:: console
 
-   brew install cmake ninja dfu-util doxygen qemu dtc python3 gperf
-   curl -O 'https://bootstrap.pypa.io/get-pip.py'
-   ./get-pip.py
-   rm get-pip.py
+   brew install cmake ninja dfu-util doxygen qemu dtc python3 python2 gperf
```
Comment (carlescufi) [inline doc/getting_started/installation_mac.rst]: Please don't include python2. macOS ships with Python 2, no need to install another copy?

Comment (carlescufi) [reply doc/getting_started/installation_mac.rst]: I will then add my comments about Python present in my PR

File: doc/getting_started/getting_started.rst [hunk 3]
```diff
@@ -231,13 +231,21 @@ without the Zephyr SDK`_ for details.
    and extract it on your file system. This example assumes the compiler was
    extracted to: :file:`<user folder>/gcc-arm-none-eabi-5_3-2016q1/`.
 
+   On macOS, the toolchain can also be installed using brew::
+
+      $ brew tap osx-cross/arm
```
Comment (dbkinder) [inline doc/getting_started/getting_started.rst]: We're avoiding use of the prompt character in code samples, making it easier to cut/paste, so delete the ``$``

File: doc/getting_started/installation_mac.rst [hunk 4]
```diff
@@ -73,26 +73,11 @@ Source :file:`zephyr-env.sh` wherever you have cloned the Zephyr Git repository:
...
-   make
-   echo "export PATH=$PWD/kconfig:\$PATH" >> $HOME/.zephyrrc
-   source $ZEPHYR_BASE/zephyr-env.sh
-
-.. note::
-
-   You only need to do this once after cloning the git repository.
-
-Finally, assuming you are using a 3rd-party toolchain you can try building the :ref:`hello_world` sample to check things out.
+Finally, assuming you are using a 3rd-party toolchain you can try building the
```
Comment (dbkinder) [inline doc/getting_started/installation_mac.rst]: ``toolchain, you can build the``

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by nashif** [comment]
obsoleted

**Pull request closed without merging**